src/rust/src/spatial_index.rs

```diff
@@ -49,13 +50,15 @@ pub fn create_cached_rtree(geoms: List) -> RTree<GeomWithData<CachedEnvelope<Geo
                 None
             } else {
                 let geo = Geom::try_from(xi).unwrap();
+                let prepared_geom: PreparedGeometry<'static> = PreparedGeometry::from(geo.geom.clone());
```

Comment:
heres' the unfortunate clone. There should be a way to avoid this, but I haven't worked that out yet.

Reply:
It looks like the AABB is still being calculated from the cached envelope of the `Geom` struct. Is there a way to get it from the prepared geometry?

Reply:
Or is it possible to completely ignore the R* tree because of the prepared geoms? 🤔